From 85ccdbb5b9c6c7a9830e5cb468662837a59a7aa3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michal=20=C4=8Ciha=C5=99?= <michal@cihar.com>
Date: Tue, 26 Jan 2016 14:25:57 +0100
Subject: [PATCH] Include common libraries in setup
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

We use PMA_fatalError which in turn needs Response and related objects.

Signed-off-by: Michal Čihař <michal@cihar.com>
---
 libraries/common.inc.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libraries/common.inc.php b/libraries/common.inc.php
index 2675b8998d3..6686cebb741 100644
--- a/libraries/common.inc.php
+++ b/libraries/common.inc.php
@@ -137,7 +137,7 @@
  */
 require './libraries/Types.class.php';
 
-if (! defined('PMA_MINIMUM_COMMON')) {
+if (! defined('PMA_MINIMUM_COMMON') || defined('PMA_SETUP')) {
     /**
      * common functions
      */
